StepperMotor/StepperMotor.c

```diff
@@ -161,15 +164,18 @@ void TIMER1_IRQHandler(void)
 	// case None defined within the StepperMotor_Step* functions when adminstration is done
 	switch(Control_GlobalStatus)
 	{
-		case Basal_Status: 
+		case Basal_Status:
+      BasalDose_TimingDisable();
 			StepperMotor_CurrentBasalDose++; // Keep track of current dosing
 			StepperMotor_StepForward();
 			break;
 		case Bolus_Status:
+      BasalDose_TimingDisable();
 			StepperMotor_CurrentBolusDose++; 
 			StepperMotor_StepForward();
 			break;
 		case Backward_Status:
+      BasalDose_TimingDisable();
```

Comment:
No, it should be in the code section that goes after it. Like before we would disable timer0 during the first iteration through the timer1 irq handler which was the steppermotor functions.